### PR TITLE
add marcokumu profile

### PR DESCRIPTION
**Pull request by marcokumu**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1542"></a>

### CODE REVIEW

**Comment by marcokumu** [comment]
LinkFree profile creation

**Comment by marcokumu** [comment]
updated checklist

**Review by schmelto — CHANGES_REQUESTED** [review]
Please revert the changes made in the package-lock.json.

**Comment by marcokumu** [comment]
I've made the requested changes. Please check.

File: package-lock.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "linkfree",
-  "version": "0.70.11",
+  "version": "0.70.1",
```
Comment (kumarsonsoff3) [inline package-lock.json:3]: You still need to make some changes! Also, remove the **empty endline** from `package-lock.json` file! 

```suggestion
  "version": "0.70.11",
```

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Not maintainable

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]

File: package-lock.json (hunk 1 above)
Comment (marcokumu) [reply package-lock.json:3]: Noted. What changes should I make?

Comment (kumarsonsoff3) [reply package-lock.json:3]: In order to solve the merge conflicts, your PR should not contain any changes in `package-lock.json` file 

Commit the changes I suggested above in line `3`, and remove the empty line from the end of the `package-lock.json` file!

Comment (marcokumu) [reply package-lock.json:3]: Thanks for the feedback. I understand the changes I'm supposed to make, however, I'm still confused about the version I'm required to use. Is it "version": "0.70.1"    or    "version": "0.70.11" ? Please elaborate so I can solve the merge conflicts.

File: package-lock.json [hunk 2]
```diff
@@ -6,7 +6,7 @@
   "packages": {
     "": {
       "name": "linkfree",
-      "version": "0.70.1",
+      "version": "0.70.11",
```
Comment (kumarsonsoff3) [inline package-lock.json:9]: we have the project version `0.70.11` and packages version `0.70.1`

```suggestion
      "version": "0.70.1",
```

**Review by kumarsonsoff3 — APPROVED** [review]
Finally we are good to go now! 🚀
Thanks for contributing! 

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Elegant, Maintainable

**Review by EmmaDawsonDev — APPROVED** [review]

**Review by Panquesito7 — APPROVED** [review]
Thanks for contributing! 🎉


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good

**Review by schmelto — APPROVED** [review]

**Review by Kartik2220 — APPROVED** [review]

**Pull request merged**